lib/js/components/Banner/Banner.vue

```diff
@@ -7,33 +7,34 @@
 					<ds-icon class="ds-banner__icon" :class="[colorClass]" :icon="icon" />
 				</div>
 
-				<div class="ds-banner__text">
-					<div class="ds-banner__title" v-text="title" />
-					<div
-						v-if="$slots.defaultText && $slots.defaultText.length > 0"
-						class="ds-banner__defaultText"
-					>
-						<slot name="defaultText" />
+				<div class="ds-banner__textWrapper">
+					<div class="ds-banner__titleWrapper">
+						<div class="ds-banner__title" v-text="title" />
+						<div
+							v-if="$slots.defaultText && $slots.defaultText.length > 0"
+							class="ds-banner__defaultText"
+						>
+							<slot name="defaultText" />
+						</div>
 					</div>
-					<div v-if="buttonText" class="ds-banner__buttonTextVertical">
-						<ds-button
-							:color="BUTTON_COLORS.NEUTRAL"
-							:type="BUTTON_TYPES.OUTLINED"
-							:size="BUTTON_SIZES.SMALL"
-							@click.native="$emit('button-clicked')"
-							>{{ buttonText }}
-						</ds-button>
+					<div class="ds-banner__rightWrapper">
+						<div v-if="buttonText" class="ds-banner__buttonText">
```

Comment:
do we need a wrapper on button? It looks that you could simplify it and move class name on Button

Reply:
If we need it should be named as other wrappers `ds-banner_buttonWrapper`

Reply:
@tweetgeek I see the wrapper is still here. DO you want to share argument behind this decision?

Reply:
its gone